src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/DisabledExternalCredsDAO.scala

```diff
@@ -0,0 +1,32 @@
+package org.broadinstitute.dsde.firecloud.dataaccess
+
+import com.typesafe.scalalogging.LazyLogging
+import org.broadinstitute.dsde.firecloud.model.{LinkedEraAccount, UserInfo, WithAccessToken}
+
+import scala.concurrent.Future
+
+class DisabledExternalCredsDAO extends ExternalCredsDAO with LazyLogging {
```

Comment:
This is needed so that Orch can run in BEEs. In the future, we'll need to get ECM to run in BEEs so that swat tests can pass without Thurloe.